README.md

```diff
@@ -44,15 +44,14 @@ TiDB Insight Scripts
 optional arguments:
   -h, --help            show this help message and exit
   -o OUTPUT, --output OUTPUT
-                        The dir to store output data of TiDB Insight, any
-                        existing file will be overwritten without futher
+                        The directory to store output data of TiDB Insight,
+                        any existing file will be overwritten without futher
```

Comment:
The directory to store the output data of TiDB Insight. **Any** existing file will be overwritten without further confirmation.